src/epics.actions.js

```diff
@@ -1,12 +1,23 @@
+const { getTeamId } = require('./team-data');
+const { describe: describeEpic, status: statusEpic } = require('./epic.actions');
+const Paginator = require('./data/paginator');
 const jiraClient = require('./jira-client');
 const { makeGetRequest } = jiraClient;
-const { getTeamId } = require('./team-data');
-const { describe: describeEpic } = require('./epic.actions');
 
 async function getEpics({ team }) {
   const id = getTeamId(team);
-  const result = await makeGetRequest(`board/${ id }/epic`);
-  const epics = result.values.filter(epic => !epic.done);
+
+  const paginator = new Paginator({
+    async fetchPage(query) {
+      return await makeGetRequest(`board/${ id }/epic`, 'agile/1.0', { query });
```

Comment:
is `agile/1.0` something for specific implementations or generic?

Reply:
I believe it is generic.